clarifai/models/model_serving/models/inference.py

```diff
@@ -10,6 +10,11 @@
 import os
 from pathlib import Path
 
+from clarifai.models.model_serving.model_config import (  # noqa # pylint: disable=unused-import
+    ModelTypes, get_model_config)
+
+#config = get_model_config("clarifai-model-type")
```

Comment:
remove this line

Reply:
It is a bit confusing that this is commented out and then replaced. Maybe leave a comment here about the behavior?

Additionally, it might be a little clearer if `"clarifai-model-type"` is replaced with something like `"MODEL_TYPE_PLACEHOLDER"`, and then the replacement code only replaces `MODEL_TYPE_PLACEHOLDER`

Reply:
The point is `inference` imported as module, so if that line is uncommented, an error will occur since `clarifai-model-type` or `MODEL_TYPE_PLACEHOLDER` is not valid model type name. A possible solution is to read `inference` as plain text instead import it, in this case can have more flexible settings.

Reply:
One additional option is that `get_model_config` can just log an error/warning when `MODEL_TYPE_PLACEHOLDER` is passed in.

If you prefer the way it currently is, that's ok, but we should definitely add a comment.

Reply:
> One additional option is that get_model_config can just log an error/warning when MODEL_TYPE_PLACEHOLDER is passed in.
Thanks for suggesting, I think this approach is the optimal choice for code clarity and safety.